-------------------------------------------------------------------------------
Test set: dominion.Randomness_ESTest
-------------------------------------------------------------------------------
Tests run: 10, Failures: 1, Errors: 3, Skipped: 0, Time elapsed: 0.024 sec <<< FAILURE! - in dominion.Randomness_ESTest
test2(dominion.Randomness_ESTest)  Time elapsed: 0.003 sec  <<< FAILURE!
java.lang.AssertionError: expected:<1> but was:<0>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at dominion.Randomness_ESTest.test2(Randomness_ESTest.java:40)

test3(dominion.Randomness_ESTest)  Time elapsed: 0.002 sec  <<< ERROR!
java.lang.SecurityException: Security manager blocks ("java.lang.RuntimePermission" "exitVM.0")
java.lang.Thread.getStackTrace(Thread.java:1556)
org.evosuite.runtime.sandbox.MSecurityManager.checkPermission(MSecurityManager.java:439)
java.lang.SecurityManager.checkExit(SecurityManager.java:761)
java.lang.Runtime.exit(Runtime.java:107)
java.lang.System.exit(System.java:971)
dominion.Randomness.main(Randomness.java:49)
dominion.Randomness_ESTest.test3(Randomness_ESTest.java:49)
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
java.lang.reflect.Method.invoke(Method.java:498)
org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
java.util.concurrent.FutureTask.run(FutureTask.java:266)
java.lang.Thread.run(Thread.java:745)

	at org.evosuite.runtime.sandbox.MSecurityManager.checkPermission(MSecurityManager.java:444)
	at java.lang.SecurityManager.checkExit(SecurityManager.java:761)
	at java.lang.Runtime.exit(Runtime.java:107)
	at java.lang.System.exit(System.java:971)
	at dominion.Randomness.main(Randomness.java:49)
	at dominion.Randomness_ESTest.test3(Randomness_ESTest.java:49)

test6(dominion.Randomness_ESTest)  Time elapsed: 0.002 sec  <<< ERROR!
java.lang.NullPointerException: null
	at dominion.Randomness.nextRandomInt(Randomness.java:34)
	at dominion.Randomness_ESTest.test6(Randomness_ESTest.java:96)

test8(dominion.Randomness_ESTest)  Time elapsed: 0.002 sec  <<< ERROR!
java.lang.NullPointerException: null
	at dominion.Randomness.nextRandomInt(Randomness.java:34)
	at dominion.Randomness.randomMember(Randomness.java:39)
	at dominion.Randomness_ESTest.test8(Randomness_ESTest.java:125)

